src/tools/ping/QuicPing.cpp

```diff
@@ -377,6 +378,13 @@ main(
         goto Error;
     }
 
+    //
+    // 0 to the server means infinite. For client, 0 will be coorced into
+    // DEFAULT_CLIENT_CONNECTION_COUNT
+    //
+    TryGetValue(argc, argv, "connections", &connections);
+    PingConfig.ConnectionCount = connections;
+
```

Comment:
Instead of doing it this way, put this code into `ParseCommonCommands` and in `ParseClientCommand` always default the count to `DEFAULT_CLIENT_CONNECTION_COUNT` before calling the function.